### PR TITLE
Update README with modern instructions to start exchangeJS locally

### PR DESCRIPTION
**Pull request by djphan**
It was difficult to start up the project as some of the instructions are old. But with this improved Readme I found it to be helpful. Let me know if this needs updating or feel free to edit per organization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,21 @@ Before trying to get started, make sure you've installed:
 
  * [Node.js](https://nodejs.org/en/download/)
  * [Yarn package manager](https://yarnpkg.com/en/docs/install)
- * [Mongodb](https://www.mongodb.com/)
+ * [Mongodb](https://www.mongodb.com/B
```
Comment (MarkBennett) [inline README.md]: Typo?

Comment (djphan) [reply README.md]: Oops. Fixed

**Comment by MarkBennett** [comment]
Looks good @djphan. Ship it! 🚢

**Comment by djphan** [comment]
Nice! I don't have the power to merge so someone needs to do it for me :d

**Comment by MarkBennett** [comment]
Missed this PR. Merging.

**Pull request merged**